src/adm/tiles.rs

```diff
@@ -113,6 +113,20 @@ impl Tile {
     }
 }
 
+pub fn filtered_dma(settings: &Settings, dma: &u16) -> String {
+    if settings
+        .excluded_dma
+        .clone()
```

Comment:
Nit: I think you could do this to save a clone.
```rs
if settings
    .excluded_dma
    .as_ref()
    .unwrap_or(&vec![])
    .contains(dma)
```

Reply:
I keep forgetting about `.as_ref()`. Thanks!